runners/samza/build.gradle

```diff
@@ -110,6 +110,8 @@ def sickbayTests = [
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testEncodingIOException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingNPException',
         'org.apache.beam.sdk.coders.PCollectionCustomCoderTest.testDecodingIOException',
+        // https://issues.apache.org/jira/browse/BEAM-6354
```

Comment:
```suggestion
        // https://github.com/apache/beam/issues/19344
```

Reply:
I know the BEAM-XYZ is the convention here, but a change to update all references is currently in flight.

Reply:
Oh, sorry about that!